packages/nimble-components/src/select/index.ts

```diff
@@ -837,6 +897,40 @@ export class Select
         this.committedSelectedOption = options[this.selectedIndex];
     }
 
+    private setActiveOption(newActiveIndex: number): void {
+        const selectedIndex = this.openActiveIndex ?? this.selectedIndex;
+        const activeOption = this.options[newActiveIndex];
+        if (isNimbleListOption(activeOption)) {
+            activeOption.activeOption = true;
+            if (selectedIndex !== newActiveIndex && selectedIndex > -1) {
+                (this.options[selectedIndex]! as ListOption).activeOption = false;
+            }
```

Comment:
Is there ever a time where the cast of `this.options[selectedIndex]! as ListOption` isn't valid? One scenario I'm thinking of is when the set of slotted options changes while the select is open. To be robust, should you check `isNimbleListOption(selectedIndex)` as part of the if-statement?

Reply:
Made the check.